as/cNFT/assembly/models/persistent_tokens_metadata.ts

```diff
@@ -0,0 +1,64 @@
+import { PersistentUnorderedMap } from 'near-sdk-as'
+import { TokenId } from '../types'
+
+@nearBindgen
+export class TokenMetadata {
+    title: string
+    issued_at: string
+    copies: u8
+    media: string
+    extra: string
+    description: string
+    media_hash: string
+    expires_at: string
+    starts_at: string
+    updated_at: string
+    reference: string
+    reference_hash: string
+}
+
+export function defaultTokenMetadata(): TokenMetadata {
+    return {
+        title: '',
+        issued_at: '',
```

Comment:
Having empty strings as default doesn't seem right, why can't it be `null`?

Reply:
Not sure why but this doesn't work (tried a long time ago too):
```
ERROR TS2322: Type '~lib/string/String | null' is not assignable to type '~lib/string/String'.
```

Could you open an issue for it actually? Time to figure out why it's not working and write it down somewhere or fix it.

Reply:
@achrafismyname @AshenI99

Reply:
Done :-)